_config/videos.yml

```
@@ -11,6 +11,66 @@
 #
 # Order videos by date descending
 
+- youtubeId: Ew6pq9nJKog
+  title: "Planning agility: continuous planning, real-time planning and more"
+  author: ge0ffrey
+  date: 2020-09-03
+  tags:
+    - insight
+
+- youtubeId: E35pYMKaukw
+  title: "Quarkus and OptaPlanner: create a school timetable application"
+  author: rsynek
+  date: 2020-09-03
+
+- youtubeId: waU8zZtx--g
+  title: "Business use cases and the impact of OptaPlanner"
+  author: satishkale
+  date: 2020-09-03
+  tags:
+    - vehicle routing
+
+- youtubeId: XCyIdmUDMtI
+  title: "Constraint Streams getting started"
+  author: triceo
+  date: 2020-09-02
+  tags:
+    - constraint
+
+- youtubeId: NHU-UZM6fKI
+  title: "Business Optimizer: I bet you I'm better than a human"
+  author: jgoldsmith613
+  date: 2020-09-02
+
+- youtubeId: 07E_v-mZ2As
+  title: "Tasks Optimization - understand the chained models"
```

```suggestion
  title: "Tasks Optimization: understand the chained models"
```

We're using colons everywhere.